blocks/roles.py

```diff
@@ -129,3 +129,10 @@ class DropoutRole(VariableRole):
 
 #: Inputs with applied dropout
 DROPOUT = DropoutRole()
+
+
+class BatchNormalizationRole(VariableRole):
```

Comment:
Just realized, since `Brick.repr(...)` will render this as `BATCH_NORMALIZATION`, we should probably change this to `BatchNormalizedRole()` (or change the constant to have name `BATCH_NORMALIZATION`, but I think `-ED` is better).